exercises/list_links.md

```diff
@@ -0,0 +1,39 @@
+## Listsof links
+ 
+
+In this exercise, we're going to consolidate our knowledge from previous sections.
```

Comment:
we don't have sections, so I would maybe leave exercises. Or "of the blocks we have learnt so far"?